src/typing.py

```diff
@@ -1899,6 +1911,22 @@ def __new__(cls, *args, **kwds):
         return _generic_new(collections.defaultdict, cls, *args, **kwds)
 
 
+if hasattr(collections, 'ChainMap'):
+    # ChainMap only exists in 3.3+
+    __all__.append('ChainMap')
+
+    class ChainMap(collections.ChainMap, MutableMapping[KT, VT],
```

Comment:
typeshed defines this as ``Dict[KT, VT]``, I am not sure this is right, but typing and typeshed should be in sync for sure.